### PR TITLE
JW7-1258 Display the display icon when paused on mobile

### PR DESCRIPTION
**Pull request by pajong**

### CODE REVIEW

File: src/css/flags/touch.less [hunk 1]
```diff
@@ -21,5 +21,8 @@
     .jw-display-icon-container {
         pointer-events: none;
     }
-}
 
+    &.jwplayer&.jw-state-paused .jw-display-icon-container {
```
Comment (donato) [inline src/css/flags/touch.less]: Why the ampersands?

Comment (pajong) [inline src/css/flags/touch.less]: because we want to select .jwplayer.jwflag-touch.jw-state-paused.
ampersands refers to parent selectors.

Comment (donato) [inline src/css/flags/touch.less]: Understood!

I was hinting that it's more complex than it probably ought to be.

If for some reason it needs to be that "specific" then the alternative is making the rule you are overwriting less "specific" instead.

Comment (donato) [inline src/css/flags/touch.less]: Another little trick that will help us in the long run is to keep track of "why" things are so specific. For example see this -> https://github.com/jwplayer/jwplayer/blob/master/src/css/flags/audioplayer.less#L33

**Pull request merged**